libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -1258,7 +1272,21 @@ void IotMqtt_Disconnect( IotMqttConnection_t mqttConnection,
 
     if( initCalled == true )
     {
-        /* Close the underlying network connection. This also cleans up keep-alive. */
+        /* Close the underlying network connection. This also cleans up keep-alive.
+         * Coverity checker USE_AFTER_FREE found a warning when trying to
+         * dereference the 'mqttConnetion' in '_IotMqtt_CloseNetworkConnection'.
+         * For this warning, freeing the pointer 'mqttConnection' was from
+         * '_IotMqtt_CreateOperation' function invoked at an earlier place in
+         * this function. In '_IotMqtt_CloseNetworkConnection', when allocation
+         * fails for the 'waitSemaphore', cleanup code will try to free
+         * 'mqttConnetion'. This will never happen as 'mqttConnetion' is
```

Comment:
`'mqttConnetion'` -> `'mqttConnection'` x3 in this paragraph

Reply:
Thanks. Will fix it.

Reply:
It's misspelled twice more in this paragraph.